### PR TITLE
Replace security proxy by gateway

### PR DESCRIPTION
**Pull request by emmdurin**
Modifier docker composition to use gateway instead of security proxy

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -162,6 +137,17 @@ services:
     environment:
       - JAVA_OPTS=-Xms512m -Xmx512m -XX:MaxPermSize=128m -Dgeorchestra.datadir=/etc/georchestra -Dgeorchestra.extensions=/mnt/mapstore_extensions -DPRINT_BASE_URL=pdf
 
+  gateway:
+    image: georchestra/gateway:latest
+    depends_on:
+      - database
+    volumes:
+      - ./config:/etc/georchestra
+      - mapstore_extensions:/mnt/mapstore_extensions
```
Comment (fvanderbiest) [inline docker-compose.yml]: mapstore is not relevant here ;-)

File: docker-compose.yml [hunk 2]
```diff
@@ -162,6 +137,17 @@ services:
...
+    environment:
+      - JAVA_OPTS=-Dgeorchestra.datadir=/etc/georchestra
+      - JAVA_OPTIONS=-Dgeorchestra.datadir=/etc/georchestra
```
Comment (fvanderbiest) [inline docker-compose.yml]: why 2 java opts ?

**Review by edevosc2c — APPROVED** [review]

**Comment by fvanderbiest** [comment]
Not to be merged before 23.0

**Comment by edevosc2c** [comment]
Will need to be rebased and add the new way of service dependencies, see https://github.com/georchestra/docker/pull/173

````yaml
  gateway:
    image: georchestra/gateway:latest
    depends_on:
      database:
        condition: service_healthy
    volumes:
      - ./config:/etc/georchestra
    environment:
      - JAVA_OPTS=-Dgeorchestra.datadir=/etc/georchestra
````

**Comment by edevosc2c** [comment]
Need a rebase against the latest changes.

**Comment by jeanmi151** [comment]
@emmdurin closing this PR because it is obsolete 
new PR : https://github.com/georchestra/docker/pull/197

**Pull request closed without merging**